src/memfs.ts

```diff
@@ -141,22 +139,16 @@ export class InMemoryFileSystem implements ts.ParseConfigHost, ts.ModuleResoluti
 	 * If there is no such file, returns undefined
 	 */
 	private readFileIfExists(path: string): string | undefined {
-
-		let content = this.overlay.get(path);
-		if (content !== undefined) {
-			return content;
-		}
-
-		const rel = path_.posix.relative('/', path);
-		content = this.overlay.get(rel);
+		const uri = util.path2uri(this.path, path);
```

Comment:
We should pass `rootUri` to the constructor as we cannot guarantee a `file://` URI